pkg/model/utils.go

```diff
@@ -36,15 +37,11 @@ func GetValidNameFromFolder(folder string) (string, error) {
 	}
 
 	name := filepath.Base(dir)
-	name = strings.ToLower(name)
```

Comment:
Info for other reviewers. We are removing this as now the name of the dev environment is the original one and we sanitize it when needed (as config map name, label values, etc...)